packages/server/src/controllers/files.ts

```diff
@@ -4,7 +4,7 @@ import fs from 'fs'
 import { promisify } from 'util'
 import { BAD_REQUEST } from 'http-status-codes'
 
-import { File, ipfs } from '@aragonone/ipfs-background-service-shared'
+import { FileMeta, ipfs } from '@aragonone/ipfs-background-service-shared'
```

Comment:
Was this rename to avoid a collision with node?

Reply:
Exactly, there is a `File` object which messes up the types.